.github/workflows/release-web.yml

```
@@ -94,15 +86,91 @@
           cname: ${{ env.TEST_ENDPOINT }}
           force_orphan: true
 
+  build-production:
+    runs-on: ubuntu-latest
+
+    if: ${{ !github.event.workflow_run || (github.event.workflow_run && github.event.workflow_run.conclusion == 'success') }}
+    steps:
+      - name: Checkout Source Code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)